### PR TITLE
[Term Entry] General / Semantic Technologies / RDF

### PR DESCRIPTION
**Pull request by noahpgordon**
### Description

Added **rdf.md** within Semantic Technologies.

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/semantic-technologies/terms/rdf.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+CatalogContent:
+  - 'paths/data-science-foundations'
+  - 'paths/data-science-nlp'
+---
+
+**Resource Description Framework (RDF)** is a language for representing information. All data within the suite of semantic technologies is represented in RDF.
+
+## RDF Triples
+
+RDF represents data in the format of **triples** -- ordered sets of three elements. The elements in an RDF triple stand for a subject, a predicate, and an object. An RDF triple can therefore be represented as follows:
```
Comment (vaibhavmani810) [inline content/general/concepts/semantic-technologies/terms/rdf.md:20]:
```suggestion
RDF represents data in the format of **triples** -- ordered sets of three elements. These elements in an RDF triple signify a subject, a predicate, and an object.
An RDF triple can therefore be represented as follows:
```

File: content/general/concepts/semantic-technologies/terms/rdf.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+```pseudo
+subject, predicate, object
+```
+
+- *`subject`*: The element which the triple says something about.
+- *`predicate`*: A feature or property that applies to `subject`.
+- *`object`*: The value of the feature or property represented by `predicate`.
+
+To illustrate, take the sentence 'The novel *Pachinko* was written by Min Jin Lee'. The sentence conveys something about *Pachinko*: that it has a feature of having an author, and that this author is Min Jin Lee. This information can be represented by the following triple:
```
Comment (vaibhavmani810) [inline content/general/concepts/semantic-technologies/terms/rdf.md:30]:
```suggestion
As an illustration, consider the sentence 'The novel Pachinko was written by Min Jin Lee.' This sentence communicates a detail about Pachinko: specifically, that it possesses a characteristic of having an author, and that this author is Min Jin Lee. This information can be depicted using the following triple:
```

File: content/general/concepts/semantic-technologies/terms/rdf.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+### Triple Terms
+
+In contrast to the above examples, the elements in a valid RDF triple cannot just be any values at all, such as arbitrary strings (e.g., `hasAuthor`). Rather, each of these elements can only be particular values:
+
+- A `subject` is either an **International Resource Identifier (IRI)**, a **blank node**, or a **quoted triple**.
+- A `predicate` must be an IRI.
+- An `object` is either an IRI, a blank node, a quoted triple, or a **literal**.
+
+Each of these terms is explained below:
```
Comment (vaibhavmani810) [inline content/general/concepts/semantic-technologies/terms/rdf.md:44]: Please add an example like you initially added while introducing triple.

File: content/general/concepts/semantic-technologies/terms/rdf.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+## RDF Serializations
+
+Valid RDF must be a written in a concrete RDF syntax, also known as an **RDF serialization**. Two important RDF serializations are briefly described below.
+
+### Turtle
+
+**Turtle** is a compact RDF syntax intended to make RDF simple for humans to read and write.
+
+The most basic way to write RDF statements in Turtle is to enclose IRIs in angle brackets and end each statement with a period, like so:
```
Comment (vaibhavmani810) [inline content/general/concepts/semantic-technologies/terms/rdf.md:59]:
```suggestion
The most basic way to write RDF statements in Turtle is to enclose IRIs in angle brackets and end each statement with a period, as shown below:
```

File: content/general/concepts/semantic-technologies/terms/rdf.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+```pseudo
+<http://books.com/books#pachinko> <http://document-ontology.org/ontology#hasAuthor> <https://www.minjinlee.com/about> .
+```
+
+The above example represents the triple about *Pachinko* using IRIs that correspond to the subject (*Pachinko*), predicate (hasAuthor), and object (Min Jin Lee).
+
+However, Turtle allows for simpler formatting of statements by using IRI prefixes and grouping predicates and objects under a common subject, like so:
```
Comment (vaibhavmani810) [inline content/general/concepts/semantic-technologies/terms/rdf.md:67]:
```suggestion
However, Turtle allows for simpler formatting of statements by using IRI prefixes and grouping predicates and objects under a common subject, as shown below:
```

**Review by vaibhavmani810 — CHANGES_REQUESTED** [review]
Hi @noahpgordon , I have some suggestions , please do the needful.

File: content/general/concepts/semantic-technologies/terms/rdf.md (hunk 3 above)
Comment (noahpgordon) [reply content/general/concepts/semantic-technologies/terms/rdf.md:46]: @vaibhavmani810 Are you asking for examples of these terms ('IRI', 'blank node', etc.), or an example of a triple using these kinds of terms?

Comment (vaibhavmani810) [reply content/general/concepts/semantic-technologies/terms/rdf.md:46]: Example of a Triple using these kind of terms.

**Review by vaibhavmani810 — CHANGES_REQUESTED** [review]

File: content/general/concepts/semantic-technologies/terms/rdf.md (hunk 2 above)
Comment (noahpgordon) [reply content/general/concepts/semantic-technologies/terms/rdf.md]: I used italics for 'Pachinko' as it is a convention to put book titles in italics.

**Comment by noahpgordon** [comment]
@vaibhavmani810 I've made relevant changes.

**Review by vaibhavmani810 — APPROVED** [review]

**Review by SSwiniarski — APPROVED** [review]
@noahpgordon Looks good.

**Pull request merged**